Dockerfile

```diff
@@ -10,6 +10,12 @@ ENV PATH $PATH:/home/node/.meteor
 COPY --chown=node package-lock.json $APP_SOURCE_DIR/
 COPY --chown=node package.json $APP_SOURCE_DIR/
 
+
+# # Install jq, a lightweight and flexible command-line JSON processor.
+# We use it to determine the base branch of a PR so we can run diffs to conditionally run circle workflows
+RUN curl -L "https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64" -o /home/node/jq
+RUN chmod +x /home/node/jq
+
```

Comment:
What do you think about removing the `jq` addition in project Dockerfile? Doesn't look like it's required. The CI script is running in the CircleCI build environment - there's no `docker run ...` in the command.

Unless you found otherwise, the script will run using the `circleci/node:8-stretch` image, as inherited from the default config. It already contains `jq`. You can verify that with 

```sh
docker pull circleci/node:8-stretch
docker run --rm circleci/node:8-stretch jq -h
```

Reply:
I did not check to see if jq was in `node:8-stretch` if that's the case, then we shouldn't need to install `jq` here. I'll remove it.